docs/sources/tempo/configuration/grafana-agent/automatic-logging.md

```diff
@@ -7,12 +7,28 @@ aliases:
 - /docs/tempo/grafana-agent/automatic-logging
 ---
 
+# The problem of trace discovery
```

Comment:
@zalegrala I'm okay with changing the page title, if you prefer, to something else. "The problem of trace discovery" is an eye-catching title, but does it describe what is on the page? If the goal of a page title is to help people identify if a page will help them learn a concept or perform a task, will the page title you've proposed help a user find the right content? 

This page is in the Configure > Grafana Agent folder because we're specifically dealing with Grafana Agent configuration. 

Whatever we rename the page, we'll need to also update the front matter (menuTitle and Title) to either match or be something similar. That will resolve the doc-validator error.

Reply:
We could change the description of the page. I'm not crazy about the second bullet (proposed) but that could help point people to TraceQL as well. 

* Current: "Automatic logging provides an easy and fast way of getting trace discovery through logs."
* Proposed: "Automatic logging provides a method of trace discovery through logs. This method is not as powerful as TraceQL."

Reply:
Oh sorry, I didn't realize this was renaming the page.  I mostly wanted to step a little slower into the problem space.    I can bring back the original title and think about what to call the other headers.

Reply:
It was a little writing que for me too, so not really neded.

Reply:
Thank you. Here's what happens: 
There is usually only one 1st level heading on a page (# heading, vs a subhead, ## heading). Docs validator expects the 1st level heading (effectively the page title) to match the title field in the front matter. (The menuTitle field can be different; it's what is displayed in the left-side navigation.) When the 1st level heading and title don't match, then it throws an error.